src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -320,129 +310,189 @@ program gnssro_bufr2ioda2
     stop 2
 endif
 
+chunksizes=(/ndata,ndata/)
```

Comment:
It can be a bit tricky finding the optimal chunk size. There are a number of things to consider, such as the impact on runtime and compression. There exists a chunk cache and one thing you don't want to happen is using a chunk size that is larger than the chunk cache - which will kill performance. This won't happen in the small ctest case, but setting the chunk size to the number of locations could cause the chunk size to exceed the chunk cache size with more realistic datasets to be run through DA flows. 

The netcdf/hdf libraries utilize a default chunking strategy if none is specified. I think for now it would be good to use the built in default and not set the chunking sizes.

However the default library chunking strategy cannot take advantage of access patterns we might be aware of that lead to better chunking strategies. I think an investigation into an optimal chunking strategy for JEDI obs data is warranted (there is an issue in ZenHub for this: #791) but that is work that isn't in the plans yet.

My recommendation is to remove the settings of chunk sizes now and investigate this at a later time. Thanks for considering this.

Reply:
Thank you for the suggestion.  chunksize is removed now.